### DISCUSSION TITLE
Multiple Windows gdscript and python

### DISCUSSION THREAD

**Discussion by mojoyup** [touilleMan/godot-python#390] [General]
I have noticed that if I write a script in python and in the same node have a script in gdscript, in my situation, I open a window to display the camera feed separate from godot window because in godot 3.6, camera feed support seems to be lacking and the godot window will not run its code but the python window will. Having both windows open, only python will execute. Godot window will only function when the python window closes. 

I cant even click in the godot window, it basically pauses.

**Comment by Zireael07** [comment]
You'd need to share your code for anyone to be able to help.
